cli/admin/src/test/java/io/pravega/cli/admin/controller/AuthEnabledControllerCommandsTest.java

```diff
@@ -0,0 +1,254 @@
+import io.pravega.cli.admin.AdminCommandState;
+import io.pravega.cli.admin.CommandArgs;
+import io.pravega.cli.admin.Parser;
+import io.pravega.cli.admin.utils.CLIControllerConfig;
+import io.pravega.cli.admin.utils.TestUtils;
+import io.pravega.client.ClientConfig;
+import io.pravega.client.admin.StreamManager;
+import io.pravega.client.connection.impl.ConnectionPool;
+import io.pravega.client.connection.impl.ConnectionPoolImpl;
+import io.pravega.client.connection.impl.SocketConnectionFactoryImpl;
+import io.pravega.client.stream.ScalingPolicy;
+import io.pravega.client.stream.StreamConfiguration;
+import io.pravega.client.stream.impl.DefaultCredentials;
+import io.pravega.common.Exceptions;
+import io.pravega.common.cluster.Host;
+import io.pravega.controller.server.SegmentHelper;
+import io.pravega.controller.store.client.StoreClientFactory;
+import io.pravega.controller.store.host.HostControllerStore;
+import io.pravega.controller.store.host.HostMonitorConfig;
+import io.pravega.controller.store.host.HostStoreFactory;
+import io.pravega.controller.store.host.impl.HostMonitorConfigImpl;
+import io.pravega.controller.util.Config;
+import lombok.Cleanup;
+import org.apache.curator.framework.CuratorFramework;
+import org.apache.curator.framework.CuratorFrameworkFactory;
+import org.apache.curator.retry.RetryOneTime;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import javax.ws.rs.core.Response;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.net.URI;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class AuthEnabledControllerCommandsTest extends AbstractAdminCommandTest {
+
+    @Before
+    @Override
+    public void setUp() throws Exception {
+        authEnabled = true;
+        super.setUp();
+    }
+
+
+    @Test
+    public void testListScopesCommand() throws Exception {
+        setupUtils.createTestStream("testListScopesCommand", 2);
+        String commandResult = TestUtils.executeCommand("controller list-scopes", state.get());
+        // Check that both the new scope and the system one exist.
+        Assert.assertTrue(commandResult.contains("_system"));
+        Assert.assertTrue(commandResult.contains(setupUtils.getScope()));
+        Assert.assertNotNull(ControllerListScopesCommand.descriptor());
+    }
+
+    @Test
+    public void testDescribeScopeCommand() throws Exception {
+        String commandResult = TestUtils.executeCommand("controller describe-scope _system", state.get());
+        Assert.assertTrue(commandResult.contains("_system"));
+        Assert.assertNotNull(ControllerDescribeStreamCommand.descriptor());
+    }
+
+    @Test
+    public void testListStreamsCommand() throws Exception {
```

Comment:
In general, you should avoid throwing the base `Exception`. If you must throw, throw the specific exceptions. Here, in the tests, throwing exceptions is meaningless, so you could simplify the code a bit by adding a `@SneakyThrows` exception, if you will.

Reply:
Using `@SneakyThrows` now.